lib/hexpm/accounts/session.ex

```diff
@@ -1,25 +1,62 @@
 defmodule Hexpm.Accounts.Session do
   use Hexpm.Schema
+  alias Hexpm.Accounts.User
 
   schema "sessions" do
-    field :token, :binary
+    belongs_to :user, User
+    field :uuid, Ecto.UUID, autogenerate: true
+    field :token_hash, :binary
     field :data, :map
+    field :expires_at, :utc_datetime_usec
+    field :active, :boolean
```

Comment:
What's the difference between `active: false` and `expires_at` being in the past?

Reply:
active false is just helpful and less error prone to check if a session is active or not vs checking expires_at, but it's not required, just a convenience.